src/components/Paddle/PaddleTransacionModal.tsx

```diff
@@ -41,14 +44,16 @@ const Error = ({ error, onClose }: { error: string; onClose: () => void }) => (
       {error}
     </AlertWithIcon>
 
-    <RecaptchaNotice />
+    {/* <RecaptchaNotice /> */}
 
     <Center>
       <Button onClick={onClose}>Close this window</Button>
     </Center>
   </Stack>
 );
 
+type CaptchaState = { status: 'success' | 'error' | 'expired' | null; token: string | null };
```

Comment:
This is what I meant that it repeats itself, if you make a component that just exports this state, we'll probably do better.